docs/code_navigation/explanations/auto_indexing_inference.mdx

```diff
@@ -16,22 +16,22 @@ For each directory containing a `go.mod` file, the following index job is schedu
 
 ```json
 {
-  "indexing_jobs": [
+  "index_jobs": [
     {
       "steps": [
         {
           "root": "<dir>",
-          "image": "sourcegraph/lsif-go",
+          "image": "sourcegraph/scip-go",
           "commands": [
             "go mod download"
           ]
         }
       ],
       "root": "<dir>",
-      "indexer": "sourcegraph/lsif-go",
+      "indexer": "sourcegraph/scip-go",
       "indexer_args": [
-        "lsif-go",
-        "--no-animation"
+        "scip-go",
+        "--animation"
```

Comment:
```suggestion
        "scip-go",
        "--verbose"
```

We don't want to recommend `--animation` as that might create way too many lines in the output (one for each "tick"), but `--verbose` is OK.

Reply:
Ah you told me this already. Will fix